example/spec/serial/AsyncOperationRunnerFailingLambdaSpec.js

```diff
@@ -20,7 +22,15 @@ describe('The AsyncOperation task runner executing a failing lambda function', (
   let failFunctionName;
   let payloadKey;
   let taskArn;
+  let originalTimeout;
 
+  beforeEach(() => {
+    originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
```

Comment:
What's the reasoning for this change?

Reply:
that's a debugging leftover from my work on this, sorry I'll remove